.github/workflows/chart-release-dispatcher.yaml

```diff
@@ -15,6 +15,7 @@ on:
       - poctagprophashedrebased
       - idonlycanary
       - idonlycanarybeta
+      - tagscanary
```

Comment:
I think this is not needed.

Reply:
We want tagscanary to be built & pushed to ghcr for canary auto-syncs.
If we remove this, auto-sync will never work

Reply:
How was it being synced before this PR? Since the code is absent. AFAIR dispatcher code is only needed in master branch. Config for image building is in another file.